README.md

```diff
@@ -518,41 +574,32 @@ document.addEventListener('pjax:send', topbar.show)
 document.addEventListener('pjax:complete', topbar.hide)
 ```
 
-### HTTP Headers
+## HTTP Headers
 
-Pjax uses several custom headers when it makes and receives HTTP requests. 
-If the requests are going to your server, you can use those headers for
-some meta information about the response.
+Pjax uses several custom headers when it makes and receives HTTP requests. If the requests are going to your server, you can use those headers for some meta information about the response.
 
-##### Request headers
+### Request Headers
 
 Pjax sends the following headers with every request:
 
 * `X-Requested-With: "XMLHttpRequest"`
 * `X-PJAX: "true"`
-* `X-PJAX-Selectors`: A serialized JSON array of selectors, taken from 
-`options.selectors`. You can use this to send back only the elements that 
-Pjax will use to switch, instead of sending the whole page. Use `JSON.parse()` 
-server-side to deserialize it back to an array.
+* `X-PJAX-Selectors`
+    * A serialized JSON array of selectors, taken from `options.selectors`. You can use this to send back only the elements that Pjax will use to switch, instead of sending the whole page. Note that you'll need to deserialize this on the server (Such as by using `JSON.parse()`)
```

Comment:
Please remove the bullet from the 2nd level.

Reply:
@BehindTheMath would you prefer it all to fall under a single bullet?

ex:

```md
* `X-PJAX-Selectors`: A serialized JSON array of selectors, taken from `options.selectors`. You can use this to send back only the elements that Pjax will use to switch, instead of sending the whole page. Note that you'll need to deserialize this on the server (Such as by using `JSON.parse()`)
```

Reply:
You can move the paragraph to the next line, just don't use a bullet. Bullets are for lists, but here there's only 1.

```
* `X-PJAX-Selectors`
  A serialized JSON array of selectors, taken from `options.selectors`.
  You can use this to send back only the elements that Pjax will use to switch, instead of sending the whole page. Note that you'll need to deserialize this on the server (Such as by using `JSON.parse()`)
```

Reply:
Fixed